### PR TITLE
Create international-journal-of-nuclear-security.csl

### PR DESCRIPTION
**Pull request by howardlhall**
CSL file for the International Journal of Nuclear Security.  The journal can be found at http://trace.tennessee.edu/ijns.  IJNS is an open access peer-reviewed journal.

### CODE REVIEW

**Comment by howardlhall** [comment]
Travis-CI failed because it did not recognize our ISSN.  Unsure why, it has been properly issued.  Please help.

**Comment by adam3smith** [comment]
no, you're just misinterpreting the message: Travis complains because you're using it twice. Just delete it as eissn -- that's only used when there are different numbers assigned to e and print edition.

**Comment by rmzelle** [comment]
(for styles in the repository, we reserve `<issn/>` for the print edition ISSN, which is always different from the electronic ISSN)

File: international-journal-of-nuclear-security.csl [hunk 1]
```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>International Journal of Nuclear Security</title>
+    <id>http://www.zotero.org/styles/international-journal-of-nuclear-security</id>
+    <link href="http://www.zotero.org/styles/international-journal-of-nuclear-security" rel="self"/>
+    <link href="http://www.sciencemag.org/about/authors/prep/res/refs.dtl" rel="documentation"/>
```
Comment (adam3smith) [inline international-journal-of-nuclear-security.csl]: do you have your own author guidelines? If so, link to those.
If you're actually just following *Science*, we'd want to make this a dependent style.

**Comment by adam3smith** [comment]
otherwise looks great. Let us know once you've update the pull request.

**Comment by howardlhall** [comment]
OK, I deleted the eissn and left that field blank, and updated the address to our instructions for authors.  Travis is running right now.

**Comment by howardlhall** [comment]
It's updated, and Travis green-lighted it.

*============================================Howard L. Hall, Ph.D.*
*Director of the UT Institute for Nuclear Security <http://nuclear.utk.edu>*
*Professor and Governor's Chair in Nuclear Security, Department of Nuclear
Engineering <http://www.engr.utk.edu/nuclear/>** and the Bredesen Center
for Interdisciplinary Research and Graduate Education  (CIRE)
<http://cire.utk.edu>*
*Senior Fellow and Director of Global Security Programs, The Howard H.
Baker Jr. Center for Public Policy <http://bakercenter.utk.edu>*
The University of Tennessee <http://www.utk.edu>
Knoxville, TN  37996-2300

Howard.Hall@utk.edu       http://about.me/Howard.Hall
*Learn more about our research group <http://howardhall.name>.*

**Comment by adam3smith** [comment]
thanks!

**Comment by rmzelle** [comment]
It doesn't matter much, but since this seems to be an online-only journal, it would have been better to delete `<issn/>` and keep `<eissn/>`. Maybe you can keep it in mind in case you submit any changes in the future.

**Comment by rmzelle** [comment]
(we are rather nitpicky when it comes to journal metadata; otherwise, thanks for contributing!!)

**Comment by howardlhall** [comment]
No problem.  Will switch it to eissn on next revision, but won't start a
revision just for that.

*============================================Howard L. Hall, Ph.D.*
*Director of the UT Institute for Nuclear Security <http://nuclear.utk.edu>*
*Professor and Governor's Chair in Nuclear Security, Department of Nuclear
Engineering <http://www.engr.utk.edu/nuclear/>** and the Bredesen Center
for Interdisciplinary Research and Graduate Education  (CIRE)
<http://cire.utk.edu>*
*Senior Fellow and Director of Global Security Programs, The Howard H.
Baker Jr. Center for Public Policy <http://bakercenter.utk.edu>*
The University of Tennessee <http://www.utk.edu>
Knoxville, TN  37996-2300

Howard.Hall@utk.edu       http://about.me/Howard.Hall
*Learn more about our research group <http://howardhall.name>.*

**Pull request merged**